docs/ui/components/PagePlatformTags/index.tsx

```diff
@@ -8,15 +8,16 @@ type Props = {
 
 export function PagePlatformTags({ platforms }: Props) {
   return (
-    <div className="mt-3 inline-flex flex-wrap">
+    <div className="mt-3 inline-flex flex-wrap gap-1">
```

Comment:
After this change tags without annotation, so on most of other pages have double spacing:

![Screenshot 2025-03-19 at 15 35 41](https://github.com/user-attachments/assets/e2638c2f-02ed-4821-8348-f986af198d40)

We should refactor how we control the spacing for all platform tag use cases, or we can try to add missing baked-in maring in those special cases tags.

Reply:
As discussed, ended up removing the tooltip as the same information is show in the tag.